website/OrderGeniusWebApp/OrderGeniusWebApp/App_Start/RazorGeneratorMvcStart.cs

```diff
@@ -0,0 +1,21 @@
+using System.Web;
+using System.Web.Mvc;
+using System.Web.WebPages;
+using RazorGenerator.Mvc;
+
+[assembly: WebActivatorEx.PostApplicationStartMethod(typeof(OrderGeniusWebApp.RazorGeneratorMvcStart), "Start")]
+
+namespace OrderGeniusWebApp {
+    public static class RazorGeneratorMvcStart {
```

Comment:
App start should be removed from the library, as it wil intefere with the sitecore pipelines.